src/main/Thumbnail.tsx

```diff
@@ -130,6 +130,7 @@ const ThumbnailTable : React.FC<{
 }> = ({inputRefs, generate, upload, uploadCallback, discard}) => {
 
   const tracks = useSelector(selectTracks)
```

Comment:
```suggestion
  const tracks = useSelector(selectVideos)
```

How about using the `selectVideos` selector instead (and maybe rename `tracks` to `videos`)? Should effectively do the same thing.